pkg/workflows/workflows_test.go

```diff
@@ -503,15 +522,15 @@ func TestCreateWorkflowFiles(t *testing.T) {
 
 	err = mockWF.CreateWorkflowFiles("helm", draftConfig, templateWriter)
 	assert.Nil(t, err)
-	os.RemoveAll(".github")
+	//os.RemoveAll(".github")
```

Comment:
nit: do these need to be deleted/uncommented?

Reply:
they should be uncommented them!